setup.py

```diff
@@ -6,7 +6,7 @@
 with open(Path(__file__).parent / "darwin" / "version" / "__init__.py", "r") as f:
     content = f.read()
     # from https://www.py4u.net/discuss/139845
-    version = re.search(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', content).group(1)
+    version = re.search(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', content).group(1)  # type: ignore
```

Comment:
Prevents mypy complaining about this - rather than refactor type, when we have to replace `setup.py` in the coming months anyway.